addons/website/static/src/interactions/text_highlights.js

```diff
@@ -8,7 +8,7 @@ import {
 } from "@website/js/text_processing";
 
 export class TextHighlight extends Interaction {
-    static selector = "#wrapwrap";
+    static selector = "#wrapwrap, .o_wslides_fs_content";
```

Comment:
I think you're fixing the fix, but not reverting it, so not a REV, and the fw-ported fix was not "useless" but "obsolete" ;)